src/Stratis.Bitcoin.Networks/BitcoinMain.cs

```diff
@@ -129,7 +129,8 @@ public BitcoinMain()
                 { 279000, new CheckpointInfo(new uint256("0x0000000000000001ae8c72a0b0c301f67e3afca10e819efa9041e458e9bd7e40"))},
                 { 295000, new CheckpointInfo(new uint256("0x00000000000000004d9b4ef50f0f9d686fd69db2e03af35a100370c64632a983"))},
                 { 486000, new CheckpointInfo(new uint256("0x000000000000000000a2a8104d61651f76c666b70754d6e9346176385f7afa24"))},
-                { 491800, new CheckpointInfo(new uint256("0x000000000000000000d80de1f855902b50941bc3a3d0f71064d9613fd3943dc4"))}
+                { 491800, new CheckpointInfo(new uint256("0x000000000000000000d80de1f855902b50941bc3a3d0f71064d9613fd3943dc4"))},
```

Comment:
That's right. Do you know why they stopped adding them?

Reply:
As far as I recall, it is because they don't really provide a useful security benefit.

If we're using them for `assumevalid` etc. to speed up validation, then that's a different matter.